JavaAssignments/src/shraddha/CalculatorSwitchCase.java

```diff
@@ -0,0 +1,65 @@
+//------------------------------------------------
+//Assignment: Calculator using Switch case.
+//------------------------------------------------
+
+package shraddha;
+
+import java.util.Scanner;
+
+public class CalculatorSwitchCase {
+
+	void calculator() {
+		Scanner sc = new Scanner(System.in);
```

Comment:
It is always better to write parameterised method , scanner class logic in main method. so that other class can also use calculator method based on their value.